src/proto/robot_status_msg.proto

```diff
@@ -10,14 +10,11 @@ message RobotStatus
     repeated ErrorCode error_code             = 2;
     BreakBeamStatus break_beam_status         = 3;
     ChipperKickerStatus chipper_kicker_status = 4;
-    DriveUnitStatus drive_units               = 5;
-    FirmwareStatus firmware_status            = 6;
-    DribblerStatus dribbler_status            = 7;
-    NetworkStatus network_status              = 8;
-    PowerStatus power_status                  = 9;
-    TemperatureStatus temperature_status      = 10;
-    Timestamp time_sent                       = 11;
-    ThunderloopStatus thunderloop_status      = 12;
+    MotorStatus motor_status                  = 5;
```

Comment:
Two things: 

1. are we following the recommended proto practice of keeping legacy proto field numbers?
2. When you were refactoring, you missed field number 7

Reply:
all the old firmware that uses this is gone, it would be good to maintain legacy proto field numbers going forward

good catch, fixed!